ledger/participant-integration-api/src/main/scala/platform/index/InMemoryStateUpdater.scala

```diff
@@ -230,6 +230,7 @@ private[platform] object InMemoryStateUpdater {
 
     TransactionLogUpdate.Transaction(
       transactionId = u.transactionId,
+      commandId = None,
```

Comment:
I understand that we filter and populate the command later, but if we add it at this level, let's add it populated.